packages/operator/pkg/appserver/workers/v1/packaging/model_packaging.go

```diff
@@ -0,0 +1,131 @@
+package packaging
+
+import (
+	"github.com/mitchellh/hashstructure"
+	odahuv1alpha1 "github.com/odahu/odahu-flow/packages/operator/api/v1alpha1"
+	packaging_types "github.com/odahu/odahu-flow/packages/operator/pkg/apis/packaging"
+	"github.com/odahu/odahu-flow/packages/operator/pkg/repository/packaging"
+	"github.com/odahu/odahu-flow/packages/operator/pkg/appserver/workers/v1/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+type KubeEntity struct {
+	obj     *packaging_types.ModelPackaging
+	storage packaging.Repository
+	service packaging.Service
+}
+
+func (k KubeEntity) GetID() string {
+	return k.obj.ID
+}
+
+func (k KubeEntity) GetSpecHash() (uint64, error) {
+	return hashstructure.Hash(k.obj.Spec, nil)
+}
+func (k KubeEntity) GetStatusHash() (uint64, error) {
+	return hashstructure.Hash(k.obj.Status, nil)
+}
+
+func (k KubeEntity) SaveResultInStorage() error {
+	return k.storage.UpdateModelPackaging(k.obj)
+}
+
+func (k KubeEntity) DeleteInService() error {
+	return k.service.DeleteModelPackaging(k.obj.ID)
+}
+
+type StorageEntity struct {
+	obj     *packaging_types.ModelPackaging
+	service packaging.Service
+}
+
+func (s *StorageEntity) GetID() string {
+	return s.obj.ID
+}
+
+func (s *StorageEntity) GetSpecHash() (uint64, error) {
+	return hashstructure.Hash(s.obj.Spec, nil)
+}
+
+func (s *StorageEntity) GetStatusHash() (uint64, error) {
+	return hashstructure.Hash(s.obj.Status, nil)
+}
+
+func (s *StorageEntity) CreateInService() error {
+	// Err then service.create
+	// If not err then calc hashes
+	// If hashes are equal do nothing
+	// If hashes are not equal then service.update
+	return s.service.CreateModelPackaging(s.obj)
+}
+
+func (s *StorageEntity) UpdateInService() error {
+	return s.service.UpdateModelPackaging(s.obj)
+}
+
+func (s *StorageEntity) DeleteInKube() error {
+	return s.service.DeleteModelPackaging(s.obj.ID)
+}
+
+type Syncer struct {
+	storage packaging.Repository
+	service packaging.Service
+}
+
+func NewSyncer(storage packaging.Repository, service packaging.Service) *Syncer {
+	return &Syncer{
+		storage: storage,
+		service: service,
+	}
+}
+
+func (mts *Syncer) AttachController(mgr ctrl.Manager, rec reconcile.Reconciler) error {
```

Comment:
Minor one - I guess mts is for Model Training Syncer. So mps should be for Model Packaging Syncer. The same is applicable to Model Deployment Syncer.
```suggestion
func (mps *Syncer) AttachController(mgr ctrl.Manager, rec reconcile.Reconciler) error {
```